packages/front-end/components/Forms/SelectField.tsx

```diff
@@ -78,6 +78,36 @@ export const ReactSelectProps = {
         color: "var(--form-multivalue-text-color)",
       };
     },
+    control: (styles) => {
```

Comment:
These changes fix a dark mode bug where the dark mode styling wasn't being applied to the `CreateableSelect` component. This fixes that bug for the pre-launch checklist, but also in other areas of the app that were using the `CreateableSelect` component (e.g. when creating a savedGroup)